linux/cuda_info.bsh

```diff
@@ -505,15 +505,16 @@ function cuda_capabilities()
       CUDA_DEPRECATED=(35 37 50)
       ;;
 
-    12.*) # 12.0-12.3
+    12.*) # 12.0-12.6
       # "SM90a or SM_90a, compute_90a – (for PTX ISA version 8.0) – adds acceleration for features like wgmma and setmaxnreg. This is required for NVIDIA CUTLASS"
       CUDA_ARCHES=(50 52 53 60 61 62 70 72 75 80 86 87 89 90a 90)
       # The 30s are gone, and there's no longer any deprecated? (50 was undeprecated?)
       CUDA_DEPRECATED=()
       ;;
 
-    # 12.4*) ?
+    # 12.7*) ?
     #   # For a while now, nvcc --help appears to have the complete list of arches finally.
+    #   echo -n 'CUDA_ARCHES=('; nvcc --help | sed -n '/^--gpu-architecture\|^--gpu-code/,/^ *$/p' | sed -n '/^ *Allowed/,/^$/p' | tr ',' '\n' | sed -E "s| *'(.*)'\\.?|\\1|; /^$/d" | grep ^sm | sort -u | sed 's|sm_||' | tr '\n' ' ' | sed 's| *$||'; echo ')'; echo -n 'CUDA_CODES=('; nvcc --help | sed -n '/^--gpu-architecture\|^--gpu-code/,/^ *$/p' | sed -n '/^ *Allowed/,/^$/p' | tr ',' '\n' | sed -E "s| *'(.*)'\\.?|\\1|; /^$/d" | grep ^compute | sort -u | sed 's|compute_||' | tr '\n' ' ' | sed 's| *$||'; echo ')'; echo -n 'CUDA_LTOS=('; nvcc --help | sed -n '/^--gpu-architecture\|^--gpu-code/,/^ *$/p' | sed -n '/^ *Allowed/,/^$/p' | tr ',' '\n' | sed -E "s| *'(.*)'\\.?|\\1|; /^$/d" | grep ^lto | sort -u | sed 's|lto_||' | tr '\n' ' ' | sed 's| *$||'; echo ')'
```

Comment:
OMG

Reply:
I've done this so many times, I just wanted something to copy and paste